manifests/resource/upstream/member.pp

```diff
@@ -39,7 +40,8 @@
   $server,
   Enum['present', 'absent'] $ensure = 'present',
   Integer $port                     = 80,
-  $upstream_fail_timeout  = '10s',
+  $upstream_fail_timeout            = '10s',
```

Comment:
Given these are new, can you use data types?

Reply:
done